tensorflow/core/lib/gtl/manual_constructor_test.cc

```diff
@@ -93,7 +93,7 @@ TEST(ManualConstructorTest, Alignment) {
   EXPECT_EQ(reinterpret_cast<char*>(test2.b.get()) - &test2.a,
             reinterpret_cast<char*>(&control2.b) - &control2.a);
 #ifdef __x86_64__
```

Comment:
We can probably remove this `#ifdef`, since the below is pretty much guaranteed to always be true.

Reply:
Hi @cantonios, yes I was testing on the CLANG compiler as we are moving to CLANG for TF Windows as well. I have removed #ifdef and the test cases are passing now for both CLANG as well as MSVC compilers

Reply:
Great, thanks for you patience.

Can you update the PR title and description to match the new change, and specify that this fixes the test for windows+clang builds?

Reply:
Thank you very much  @cantonios for your guidance. I have made the changes.